scene/3d/gpu_particles_3d.cpp

```diff
@@ -348,7 +348,7 @@ void GPUParticles3D::_notification(int p_what) {
 		}
 	}
 
-	// Use internal process when emitting and one_shot are on so that when
+	// Use internal process when emitting and one_shot is on so that when
```

Comment:
"when A and B **are** on", that was correct.

Reply:
Maybe change the order, so it is not confused with "when emitting, and one shot is on"?

Reply:
Actually reading the code, "is" could be considered correct too. It's kind of the same meaning in the end (since the `emitting` property being on is the same as the node emitting).